docs/user_guide/index.md

```diff
@@ -0,0 +1,87 @@
+# User guide
+
+```{toctree}
+:hidden:
+v3 to v4 migration guide <v4-migration>
+Example scripts <additional_examples>
+Community examples <https://github.com/Parcels-code/parcels_contributions>
```

Comment:
Given the current maintenance status of `parcels_contributions`, I don't think that we should promote this here in the sidebar. I think its important to distinguish it as a separate project - happy to just reference it in the community section.